ui/src/app/services/test-step.service.ts

```diff
@@ -90,6 +90,16 @@ export class TestStepService {
     );
   }
 
+  public checkForLinkedTestSteps(steps:TestStep[]): Observable<Page<TestStep>>{
```

Comment:
seems like same fun name which solve different purpose. Plz try to make it appropriate